torchao/experimental/kernels/cpu/aarch64/kleidi/kai_matmul_clamp_f32_qai8dxp1x8_qsi4c32p4x8_1x4x32_neon_dotprod.h

```diff
@@ -94,7 +94,6 @@ void kernel(
     const void* activation_data,
     float clamp_min,
     float clamp_max) {
-  assert(output_m_stride == n);
```

Comment:
why remove?

Reply:
this was incorrect because the meaning of the argument `n` is not same a `N`. Kleidi should do `output_m_stride == N (not n)` issert internally as well.